sphinxcontrib/test_reports/directives/test_case.py

```diff
@@ -129,6 +131,16 @@ def run(self, nested=False, suite_count=-1, case_count=-1):
 """.format("\n   ".join([x.lstrip() for x in case["system-out"].split("\n")]))
 
         time = case["time"]
+        # Ensure time is a string, SN 6.0.0 requires
+        # to be in one specific type
+        # Handle time conversion if it's a number (seconds)
+        if isinstance(time, (int, float)) and time > 0:
```

Comment:
This logic looks brittle. Time is defined as string, without a datetime format. So if the time cannot be converted, I would just put an empty string. Or something in the dark ages. `now` could be valid data.